### ISSUE TITLE
OneDrive config does not work since at least rclone 1.44

### ISSUE THREAD

**Issue by x0b** [kaczmarkiewiczp/rcloneExplorer#246]
**TLDR:** OneDrive config does not work, no easy fix in sight. Workaround by importing a good config.

---

Since at least v1.44, rclone requires additional parameters for OneDrive remotes - namely ```drive_type``` and ```drive_id```. Remotes configured with the rcloneExplorer dialog lack these options and are as such non-functional. 

The values for these configuration options are nontrivial - especially ```drive_id``` is a longer hex string that a user normally does not know. They are usually discovered using the interactive remote config dialog, which does not work on rcloneExplorer. Importing a OneDrive config works just fine - the values for these options don't change. 

I don't really see a quick solution here - unless a special or generic interactive configuration logic is added, this remote will not be configurable on device.
